python/pyspark/cloudpickle.py

```diff
@@ -42,27 +42,42 @@
 """
```

Comment:
@codesue, we should make it as a proper module so cloudpickle_fast can be properly used as well. Seems like v.1.5 was release too .. we should better match it as well.

Let me create a PR by myself to upgrade to 1.5, and co-author you together.

Reply:
That sounds great! Thank you!!